ui/components/MeshConfigComponent.js

```diff
@@ -147,10 +149,16 @@ class MeshConfigComponent extends React.Component {
       operatorVersion : "N/A",
       meshSyncInstalled : false,
       meshSyncVersion : "N/A",
-      NATSInstalled : false,
+      NATSState : "UNKNOWN",
```

Comment:
Gents, let's avoid all caps in the UI, unless the text is a label.

Reply:
Ok